crates/web-sys/webidls/enabled/Element.webidl

```diff
@@ -263,3 +263,18 @@ partial interface Element {
   [NeedsCallerType]
   undefined requestPointerLock();
 };
+
+interface mixin HTMLOrForeignElement {
+  [SameObject] readonly attribute DOMStringMap dataset;
+
+  [CEReactions, SetterThrows, Pure] attribute boolean autofocus;
+  [CEReactions, SetterThrows, Pure] attribute long tabIndex;
+  [Throws, NeedsCallerType] undefined focus(optional FocusOptions options = {});
```

Comment:
```suggestion
  attribute DOMString nonce; // intentionally no [CEReactions]

  [CEReactions, SetterThrows] attribute boolean autofocus;
  [CEReactions, SetterThrows] attribute long tabIndex;
  [Throws] undefined focus(optional FocusOptions options = {});
```
Lets follow the specification as close as possible here.
(also adding the new `nonce` attribute)